db/migrations/20240529151056_Page.sql

```diff
@@ -0,0 +1,9 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Page (
+    page_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    user_id Uuid UNIQUE,
+    url_id Uuid UNIQUE,
+    url VARCHAR(255) NOT NULL
+);
```

Comment:
Nit: Can we add a newline at the end of this file?

Here is a way to make VS Code do this automatically so you don't have to worry about it!
https://stackoverflow.com/a/44704969